Artifacts/windows-vscode/installVSCode.ps1

```diff
@@ -0,0 +1,36 @@
+﻿Function Get-RedirectedUrl {
```

Comment:
Do consider using $ErrorActionPreference = "stop". This will stop execution at first failure.

Also consider returning a non-zero exit code in case of failure. See example: https://github.com/Azure/azure-devtestlab/blob/master/Artifacts/windows-enable-local-admins/EnableLocalAdmins.ps1